### ISSUE TITLE
Some questions of evaluation.

### ISSUE THREAD

**Issue by opjang5** [Zengyi-Qin/Weakly-Supervised-3D-Object-Detection#11]
Hello, I'm very interested in your wso3d project. Did you use AP_11 in your evaluation or AP_40? I reproduced your model and have found AP_11 score is more similar to your result in your paper.

**Comment by Zengyi-Qin** [comment]
Yes 11 recall positions are used to calculate the AP. The issue is also clarified in https://github.com/Zengyi-Qin/Weakly-Supervised-3D-Object-Detection/issues/8
